packages/vue/src/components/list/MultiDropdownList.jsx

```diff
@@ -257,7 +280,7 @@ const MultiDropdownList = {
 				finalValues = value;
 				currentValue = {};
 
-				if (value) {
+				if (value && value.length) {
```

Comment:
Why there is a length check?
Are you trying to make sure that the value is an `Array`?
If yes then use `Array.isArray()` instead.